source/extensions/common/wasm/null/wasm_api_impl.h

```diff
@@ -73,6 +73,17 @@ inline void proxy_setMetadataStruct(MetadataType type, const char* name_ptr, siz
 // Continue
 inline void proxy_continueRequest() { continueRequestHandler(current_context_); }
 inline void proxy_continueResponse() { continueResponseHandler(current_context_); }
+inline void proxy_sendLocalReply(uint32_t response_code, const char* response_code_details_ptr,
```

Comment:
Nit: I'd prefer if this was called `sendLocalResponse`.

Reply:
I did this to match the envoy name, but I agree, that is a better name.

Reply:
We don't match Envoy names for most part (e.g. request/response vs encode/decode), so I don't see a strong need to do it here either.

Reply:
Changed.  I almost just called it "respond" but that is a bit terse.